cmd/poold/main.go

```diff
@@ -77,7 +77,7 @@ func start() error {
 		}
 
 		go func() {
-			listenAddr := net.JoinHostPort("", config.Profile)
+			listenAddr := net.JoinHostPort("127.0.0.1", config.Profile)
```

Comment:
there are very valid use case for allowing profiling on a externally reachable port though.  Could we make the `config.Profile` option optionally set this IP?